### ISSUE TITLE
The classification accuracy of the flower dataset is low

### ISSUE THREAD

**Issue by lucy11111111** [THU-MIG/RepViT#18]
![QQ图片20231007101512](https://github.com/THU-MIG/RepViT/assets/116077427/c063d021-e6cc-4843-aa20-2773c9834687)
I used the flower classification dataset from this URL for training：[https://www.robots.ox.ac.uk/~vgg/data/flowers/102/](url), but I got low accuracy without modifying any parameter of the original network, may I ask why this? I used some other dataset for training and also got the same low accuracy.

**Comment by jameslahm** [comment]
Did you load the pre-trained checkpoint?

**Comment by lucy11111111** [comment]
Isn't this a pre-trained weight in the original code?
parser.add_argument('--teacher-path', type=str,
                        default='https://dl.fbaipublicfiles.com/deit/regnety_160-a5fe301d.pth')

**Comment by jameslahm** [comment]
No, this is the pre-trained weight of the teacher model. I mean the pre-trained weight of the student model on ImageNet.

**Comment by lucy11111111** [comment]
I see, is it easy to give a download link for the pre-trained weights? Can you tell me exactly how this weight is put into the code?

**Comment by jameslahm** [comment]
The download links for the pre-trained weights are here: https://github.com/THU-MIG/RepViT#models

**Comment by lucy11111111** [comment]
May I ask where this weight is loaded in the code? Where do I need to change the code?

**Comment by jameslahm** [comment]
You could refer to https://pytorch.org/tutorials/beginner/saving_loading_models.html.
